include/NovelRT/Maths/GeoMatrix4x4F.h

```diff
@@ -22,139 +25,726 @@ namespace NovelRT::Maths
+         *      \begin{bmatrix}
+         *      1 & 7 & 6 & 5\\
+         *      2 & 1 & 4 & 3\\
+         *      3 & 5 & 1 & 2\\
+         *      4 & 6 & 7 & 1
+         *      \end{bmatrix}
+         *      + 6
+         *      \rightarrow
+         *      \begin{bmatrix}
+         *      1 & 7 & 6 & 5\\
+         *      2 & 1 & 4 & 3\\
+         *      3 & 5 & 1 & 2\\
+         *      4 & 6 & 7 & 1
+         *      \end{bmatrix}
+         *      +
+         *      \begin{bmatrix}
+         *      6 & 6 & 6 & 6\\
+         *      6 & 6 & 6 & 6\\
+         *      6 & 6 & 6 & 6\\
+         *      6 & 6 & 6 & 6
+         *      \end{bmatrix}
+         *      =
+         *      \begin{bmatrix}
+         *      1+6 & 7+6 & 6+6 & 5+6\\
+         *      2+6 & 1+6 & 4+6 & 3+6\\
+         *      3+6 & 5+6 & 1+6 & 2+6\\
+         *      4+6 & 6+6 & 7+6 & 1+6\\
+         *      \end{bmatrix}
+         *      \rightarrow
+         *      \begin{bmatrix}
+         *      7 & 13 & 12 & 11\\
+         *      8 & 7 & 10 & 9\\
+         *      9 & 11 & 7 & 8\\
+         *      10& 12 & 13 & 7
+         *      \end{bmatrix}
+         * \f]
+         * @param other The other scalar that gets added together with this matrix.
+         * @return A matrix who's components consist of the entrywise sum of this and another scalar.
+         */
         inline GeoMatrix4x4F operator+(float other) const noexcept
         {
             return GeoMatrix4x4F(*reinterpret_cast<const glm::mat4*>(this) + other);
         }
 
+        /**
+         * @brief Constructs a matrix containing the entrywise difference of this and another scalar.
+         *
+         * @details
+         * An example of a matrix that would be constructed from an entrywise difference with a scalar: \f[
+         *      \begin{bmatrix}
+         *      1 & 7 & 6 & 5\\
+         *      2 & 1 & 4 & 3\\
+         *      3 & 5 & 1 & 2\\
+         *      4 & 6 & 7 & 1
+         *      \end{bmatrix}
+         *      -6
+         *      \rightarrow
+         *      \begin{bmatrix}
+         *      1 & 7 & 6 & 5\\
+         *      2 & 1 & 4 & 3\\
+         *      3 & 5 & 1 & 2\\
+         *      4 & 6 & 7 & 1
+         *      \end{bmatrix}
+         *      -
+         *      \begin{bmatrix}
+         *      6 & 6 & 6 & 6\\
+         *      6 & 6 & 6 & 6\\
+         *      6 & 6 & 6 & 6\\
+         *      6 & 6 & 6 & 6
+         *      \end{bmatrix}
+         *      =
+         *      \begin{bmatrix}
+         *      1-6 & 7-6 & 6-6 & 5-6\\
+         *      2-6 & 1-6 & 4-6 & 3-6\\
+         *      3-6 & 5-6 & 1-6 & 2-6\\
+         *      4-6 & 6-6 & 7-6 & 1-6\\
+         *      \end{bmatrix}
+         *      \rightarrow
+         *      \begin{bmatrix}
+         *      -5 & 1 & 0 & -1\\
+         *      -4 & -5 & -2 & -3\\
+         *      -3 & -1 & -5 & -4\\
+         *      -2 & 0 & 1 & -5\\
+         *      \end{bmatrix}
+         * \f]
```

Comment:
As per my other question, is this correct?

Reply:
after testing it works like expected